libevmasm/LinkerObject.cpp

```diff
@@ -38,7 +38,7 @@ void LinkerObject::link(map<string, h160> const& _libraryAddresses)
 	std::map<size_t, std::string> remainingRefs;
 	for (auto const& linkRef: linkReferences)
 		if (h160 const* address = matchLibrary(linkRef.second, _libraryAddresses))
-			address->ref().copyTo(ref(bytecode).cropped(linkRef.first, 20));
+			copy(address->data(), address->data() + 20, bytecode.begin() + linkRef.first);
```

Comment:
I assume there is a test for this?

Reply:
No special test for his, but the end-to-end-tests make heavy use of the linker.